frontend/src/views/HomeView.vue

```diff
@@ -103,40 +103,11 @@ const { getConfig } = storeToRefs(useConfigStore());
         Versioning, metadata, tagging, syncing with existing buckets and more
       </h1>
     </div>
-    <div
-      class="flex justify-content-center mb-3"
-      style="height: 32px"
-    >
-      <p class="text-xl">
-        We continue to expand the features of BCBox. Stay tuned for documentation on our <a
-          target="_blank"
-          href="https://github.com/bcgov/bcbox/wiki"
-        >wiki</a>.<br /><br />
-      </p>
-    </div>
     <div class="flex justify-content-center mb-8 text-xl">
       <ul>
         <li>
-          Need a bucket? Contact <a href="mailto: NRIDS.Optimize@gov.bc.ca">NRIDS Optimization</a>
-          (Natural Resource ministries) or your <br /> ministry's service desk
-        </li>
-        <li>
-          See BCBox on
-          <a
-            href="https://github.com/bcgov/bcbox"
-            target="_blank"
-          >Github</a>
-        </li>
-        <li>
-          Learn more about the
-          <a
-            href="https://bcgov.github.io/common-service-showcase/"
-            target="_blank"
-          >Common Services Showcase Team</a>
-        </li>
-        <li>
-          Email the <a href="mailto: nrm.commonserviceshowcase@gov.bc.ca">Common Services Showcase Team</a>
-          with questions or feedback
+          Need a bucket? Contact <a href="https://apps.nrs.gov.bc.ca/int/jira/servicedesk/customer/portal/1/create/701">NRIDS Optimization</a>
+          (Natural Resource ministries) or your ministry's service desk – you will need a bucket to get started
```

Comment:
The wording here seems redundant. May be worth just saying...
```txt
Contact NRIDS Optimization or your ministry's service desk. You will need a bucket to get started.
```
Without the need a bucket rhetorical. Also link here should have a `target="_blank"` to be consistent with all our other links.